org.openjdk.jmc.console.ext.agent/src/main/java/org/openjdk/jmc/console/ext/agent/ui/LoadAgentSection.java

```diff
@@ -118,22 +123,32 @@ public void handleEvent(Event event) {
 				}
 			}
 		});
+
+		error = new GenericErrorMessage(this);
+		error.setVisible(false);
 	}
 
 	public void setLoadAgentListener(Runnable listener) {
 		this.loadAgentListener = listener;
 	}
 
 	private boolean loadAgent(String agentJar, String xmlPath) {
+		if (Files.notExists(Path.of(agentJar, ""))) {
```

Comment:
`Path.of()` is introduced in Java 11. Please mind compatibility and use `Paths.get()` instead.

Reply:
fixed in https://github.com/rh-jmc-team/jmc-agent-plugin/pull/27/commits/55bd0ce75dc38e3bd0665569d7a347f40fc2a20c